EnglishTranslation-master/UI.tsv

```diff
@@ -2615,7 +2615,7 @@ UI_20170330_002614	{@st45tw2}Others{/}	{@st45tw2}기타{/}
 UI_20170330_002615	{@st42}Join	{@st42}참석하기				
 UI_20170330_002616	{@st42}Close	{@st42}닫기				
 UI_20170330_002617	{@st59}Close Message Box{/}	{@st59}메시지 함을 닫습니다{/}	UI_20150317_000017	{@st59}Close{/}	{@st59}경매장을 닫습니다{/}	
-UI_20170330_002618	{#62100c}Untradable items: %s	{#62100c}거래불가 개수 : %s개	UI_20150918_001804	Total Untradable: %s	총 거래 불가: %s개	
+UI_20170330_002618	{#62100c}Untradables: %s	{#62100c}거래불가 개수 : %s개	UI_20150918_001804	Total Untradable: %s	총 거래 불가: %s개	
```

Comment:
This was fine as is, and always has been; untradeables is bad use of plural,  at best the I in items should be capitalized and that's it

Reply:
👍